### ISSUE TITLE
Wrong terminal position if it has no Menubar

### ISSUE THREAD

**Issue by foxmadr** [johannesjo/gnome-shell-extension-window-session-manager#19]
The position all terminals is wrong if Menubar is not visible for them. I have move them up manually after each reboot.
Tested on Ubuntu 16 and 18.

**Comment by johannesjo** [comment]
Are they slightly off or completely? 

I also have an app or two which misbehave when they are not full height and at the top of the screen. Not sure if this is fixable on my end or if it is an issue with the x window system.

**Comment by foxmadr** [comment]
Only vertical (Y) coordinate is wrong. X position is correct. It seems to me like the algorithm is calculating with the height of menu bar, but the menu bar is hidden for my terminal windows (to get more space on the desktop).
And it happens only to terminal windows. Others are positioned correctly, e.g. Chrome, Firefox.

**Comment by johannesjo** [comment]
Yes I had the same feeling. I even tried calculating the position further up north by exactly that amount, but that didn't do anything. There seems to be going on something with the X-Window-System that I don't quite understand and unfortunately the documentation I found so far is very bad. I [posted a question on redit](https://reddit.com/r/linuxquestions/comments/e3hluc/is_there_anybody_here_familiar_with_the_x_window/) maybe somebody else can help.

**Comment by foxmadr** [comment]
I see. Thanks

pá 29. 11. 2019 v 21:44 odesílatel Johannes Millan <notifications@github.com>
napsal:
